tests/ballerina-unit-test/src/test/resources/test-src/types/map-struct-json-unified.bal

```diff
@@ -35,15 +35,15 @@ function testAccessJsonInStruct () returns (string, string, string) | error {
 
     var result1 = p1.parent;
     if (result1 is Person) {
-        status1 = check <string>result1.info.status;
+        status1 = check string.create(result1.info.status);
     } else {
         io:println("Person is null");
     }
 
     var result2 = p1["parent"];
     if (result2 is Person) {
-        status2 = check <string> result2["info"]["status"];
-        status3 = check <string> result2.info["status"];
+        status2 = check string.create(result2["info"]["status"]);
+        status3 = check string.create(result2.info["status"]);
```

Comment:
Noticed that in some places check is used, and in others, not. Is that intentional? is create() safe in some instances?

Reply:
Yeah, like the safe and unsafe conversions we used to have.